src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -47,6 +47,12 @@
     private static final Path SHALLOW_CLONING_FLAG_OVERRIDE_TEST_CONFIG_FILES =
             loadResource(RepoConfigurationTest.class,
             "RepoConfigurationTest/repoconfig_shallowCloningOverrideCsv_test");
+    private static final Path FIND_PREVIOUS_AUTHORS_TEST_CONFIG_FILES =
+            loadResource(RepoConfigurationTest.class,
+                    "RepoConfigurationTest/repoconfig_findPreviousAuthors_test");
+    private static final Path FIND_PREVIOUS_AUTHORS_FLAG_OVERRIDE_TEST_CONFIG_FILES =
+            loadResource(RepoConfigurationTest.class,
+                    "RepoConfigurationTest/repoconfig_findPreviousAuthorsOverrideCsv_test");
```

Comment:
The extra spaces are not necessary.

Reply:
![Screenshot 2021-09-06 at 12 14 52 AM](https://user-images.githubusercontent.com/61911161/132133804-eae3e660-4bac-48c5-abe9-f79b0936d483.png)
For spaces are you referring to this?

Reply:
No sorry, I meant that the lines 54 and 55 should be aligned, so total 12 spaces in front.

Reply:
ah ok, thanks for the clarification